### PR TITLE
Update Tips resolver

### PR DESCRIPTION
**Pull request by ishrak-imam**
Update tips resolvers to get the data from the squid indexer.

### CODE REVIEW

File: subschemas/substrate-chain/src/typeDefs/tips.ts [hunk 1]
```diff
@@ -9,11 +8,13 @@ export default /* GraphQL */ `
...
-    finder: AccountInfo
+    who: Account!
+    finder: Account
     reason: String!
     deposit: String
+    formattedDeposit: String
     closes: String
+    createdAt: String!
     median: String
     formattedMedian: String
```
Comment (jdomingos) [inline subschemas/substrate-chain/src/typeDefs/tips.ts:19]: These field names are not really explanatory. 
But I know it doesn't belong to this PR - I was just thinking about naming here 😅  `</rant>`

**Review by jdomingos — APPROVED** [review]
Nice, looks good - two comments but really actionable for this PR and definitively not blocking 👍

File: subschemas/substrate-chain/src/typeDefs/tips.ts (hunk 1 above)
Comment (ishrak-imam) [reply subschemas/substrate-chain/src/typeDefs/tips.ts]: I have changed the field names. 👍

File: graphql-server/src/remoteSchemas.ts [hunk 2]
```diff
@@ -11,10 +11,6 @@ export default [
     name: 'substrate-governance',
     url: 'https://squid.litentry.io/governance/graphql',
   },
-  {
-    name: 'substrate-tips',
-    url: 'https://squid.litentry.io/tips/graphql',
-  },
```
[the remark this answers is not in the document]
Comment (ishrak-imam) [reply graphql-server/src/remoteSchemas.ts:17]: Should I put it back?
